go/vt/vtgr/controller/group.go

```diff
@@ -31,22 +33,31 @@ import (
 var (
 	groupOnlineSize = stats.NewGaugesWithMultiLabels("MysqlGroupOnlineSize", "Online MySQL server in the group", []string{"Keyspace", "Shard"})
 	isLostQuorum    = stats.NewGaugesWithMultiLabels("MysqlGroupLostQuorum", "If MySQL group lost quorum", []string{"Keyspace", "Shard"})
+
+	heartbeatThreshold = flag.Int("group_heartbeat_threshold", 0, "Group heartbeat staleness threshold. Need to set together with -enable_heartbeat_check")
 )
 
 // SQLGroup contains views from all the nodes within the shard
 type SQLGroup struct {
-	views         []*db.GroupView
-	resolvedView  *ResolvedView
-	logger        *log.Logger
-	size          int
-	singlePrimary bool
-	statsTags     []string
+	views                       []*db.GroupView
+	resolvedView                *ResolvedView
+	logger                      *log.Logger
+	size                        int
+	singlePrimary               bool
+	heartbeatStalenessThreshold int
```

Comment:
Not a big fan of long field names, but if this is required to express the meaning, that's fine.

Reply:
i renamed it to `heartbeatThreshold`, i can change it back if it feels less clear